arm-machinelearning/2017-01-01/swagger/webservices.json

```diff
@@ -0,0 +1,1105 @@
+            "description": "Success. The response includes a paginated array of web service objects and a URI to the next set of results, if any. For the more information the limits of the number of items in a resource group, see https://azure.microsoft.com/en-us/documentation/articles/azure-subscription-service-limits/. Note that the web service objects are sparsely populated to conserve space in the response content. To get the full web service object, call the GET operation on the web service.",
+            "schema": {
+              "$ref": "#/definitions/PaginatedWebServicesList"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearning/webServices": {
+      "get": {
+        "tags": [
+          "WebServices"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "operationId": "WebServices_List",
+        "description": "Gets the web services in the specified subscription.",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "$skiptoken",
+            "type": "string",
+            "description": "Continuation token for pagination.",
+            "required": false
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The response includes a paginated array of web service objects and a URI to the next set of results, if any. Note that the web service objects are sparsely populated to conserve space in the response content. To get the full web service object, call the GET operation on the web service.",
+            "schema": {
+              "$ref": "#/definitions/PaginatedWebServicesList"
+            }
+          }
+        }
+      }
+    }    
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "description": "The Azure subscription ID.",
+      "in": "path",
+      "type": "string",
+      "required": true
+    },
+    "ResourceGroupNameParameter": {
+      "name": "resourceGroupName",
+      "description": "Name of the resource group in which the web service is located.",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "WebServiceNameParameter": {
+      "name": "webServiceName",
+      "description": "The name of the web service.",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "APIVersionParameter": {
+      "in": "query",
+      "name": "api-version",
+      "type": "string",
+      "description": "The version of the Microsoft.MachineLearning resource provider API to use.",
+      "required": true
+    }
+  },
+  "definitions": {
+    "Resource": {
```

Comment:
The definition should have "description".

From https://travis-ci.org/Azure/azure-rest-api-specs/jobs/209869904#L1172